util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -1086,6 +1098,46 @@ static bool HashMCTSha3(const Span<const uint8_t> args[],
       {Span<const uint8_t>(md[1000])});
 }
 
+static unsigned char* BuildLDTMessage(const bssl::Span<const uint8_t> part_msg, int times) {
```

Comment:
Do you have a link to where this test is defined and how the message should be constructed?